recipes/odl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+    name: odl
+    version: "0.8.0"
```

Comment:
Please template this using Jinja or otherwise the conda-forge bots won't work.

Reply:
Done in [6cf601f](https://github.com/conda-forge/staged-recipes/pull/27504/commits/6cf601f899fc42f5f6fd70645987e1fe226a1b60)